Sources/Fuzzilli/Core/JavaScriptEnvironment.swift

```diff
@@ -497,7 +522,7 @@ public extension ObjectGroup {
             "lastIndexOf" : [.anything, .opt(.integer)] => .integer,
             "match"       : [.regexp] => .jsString,
             "matchAll"    : [.regexp] => .jsString,
-            //"normalize"   : [.string] => .jsString),
+            "normalize"   : [.opt(.string)] => .jsString,        //@@ why??
```

Comment:
I'll merge it into this PR later today.

Reply:
I've added the StringNormalizeGenerator. It wasn't obvious where to put it, so I added it by the `TemplateStringGenerator`.

It looks like `ArrayBuffer.prototype.resize` was added to Fuzzilli by/for V8. The [V8Profile](https://github.com/phoddie/fuzzilli/blob/d5e70661f57f9c7167a8e72bb2a387e9b91df201/Sources/FuzzilliCli/Profiles/V8Profile.swift#L32-L45) has a code generator for that and our branch of XSProfile has [several](https://github.com/phoddie/fuzzilli/blob/810dacae8df60b4478875b1faf0969ad0ee0b20a/Sources/FuzzilliCli/Profiles/XSProfile.swift#L103-L129) (slightly messy...) [generators](https://github.com/phoddie/fuzzilli/blob/810dacae8df60b4478875b1faf0969ad0ee0b20a/Sources/FuzzilliCli/Profiles/XSProfile.swift#L37-L93) around that. It seems likely that both XS & V8 added generators because Fuzzilli won't quickly discover the options object to the constructor, `{maxByteLength: 128}`, needed to create  a resizable buffer. Would it make sense to add build that into Fuzzilli as well? A simple start would be to use the V8 variation with an update-to-date list of TypedArray constructors. If you agree, I can update this PR with that as well.

Reply:
Cool, thanks! Then let's also turn this into `[] => .jsString,      // the first parameter must be a specific string value, so we have a CodeGenerator for that instead.`

Re. the CodeGenerators for the resizable/growable ArrayBuffers: so there will soon be a new mutator that will be able to figure out that `maxByteLength` is interesting and will create objects with that property. However, it probably still makes sense to have dedicated CodeGenerators for them. WDYT about adding two CodeGenerators: `ResizableArrayBufferGenerator` and `GrowableSharedArrayBufferGenerator` that basically do what the one in the V8Profile does (and deleting that one): create the ArrayBuffer and wrap a TypedArray around it?

Reply:
> there will soon be a new mutator that will be able to figure out that maxByteLength is interesting and will create objects with that property

Very cool. So a way to find the properties used in an options object?

> WDYT about adding two CodeGenerators: ResizableArrayBufferGenerator and GrowableSharedArrayBufferGenerator that basically do what the one in the V8Profile does (and deleting that one): create the ArrayBuffer and wrap a TypedArray around it?

Agreed. I'll do that.

FWIW – to be entirely parallel, there should also be generators for a DataView wrapped around both kinds of resizable buffers. Worth doing?

Reply:
> Very cool. So a way to find the properties used in an options object?

The new mutator tries to find properties that are accessed on an object but don't exist (by using Proxies). That way, it'll sooner or later discover that something used as 2nd argument to the `ArrayBuffer` constructor will have its `maxByteLength` read. It'll then add that property.

> FWIW – to be entirely parallel, there should also be generators for a DataView wrapped around both kinds of resizable buffers. Worth doing?

I don't think it's really worth adding `DataView` generators as well. We also don't have "normal" DataView code generators. They'll eventually be constructed from other mutations/code generations, and if they are interesting enough (trigger a lot of new coverage), they'll be used more frequently. So I think that's probably good enough.

Reply:
Nice idea to use a Proxy to see what properties are being accessed. With options objects being so widely used, that will surely uncover a few surprises.

I updated the PR as requested. In making the change, I noticed that we could simply add `DataView` to the `TypedArray` list in both `ResizableArrayBufferGenerator` and `GrowableSharedArrayBufferGenerator`. That would be a low-overhead way to get some coverage there. No problem if you'd prefer not to.

Reply:
> I noticed that we could simply add DataView to the TypedArray list in both ResizableArrayBufferGenerator and GrowableSharedArrayBufferGenerator

Yeah that's a nice idea! Let's go with that!

Reply:
Done!